modules/localization/conf/localization.conf

```diff
@@ -211,3 +211,7 @@
 # need cpu to support avx intel intrinsics
 # default: false
 --if_use_avx=true
+
+--lidar_extrinsics_file=/apollo/modules/localization/msf/params/velodyne_params/velodyne64_novatel_extrinsics_example.yaml
+--map_dir=/apollo/Bag/Test_Data/Sunnyvale_Big_Loop
```

Comment:
This `map_dir` should be overwritten or mismatch with the ones we selected from dreamview frontend.